api/staleness.py

```diff
@@ -58,6 +58,11 @@ def _validate_input_data(body):
     return validated_data
 
 
+def _get_org_id():
+    current_identity = get_current_identity()
+    return current_identity.org_id
```

Comment:
Since current_identity is only used once, this can just be a one-liner
```suggestion
    return get_current_identity().org_id
```

Reply:
I refactored using your suggestion